### ISSUE TITLE
Don't have uncertainty info in edges

### ISSUE THREAD

**Issue by leilai125** [moloydas/pose_graph_optimization_python#1]
Hi, if my data doesn't have uncertainty info (the last 6 numbers), can I still use this algorithm?

**Comment by moloydas** [comment]
No, it needs uncertainty information. if your data stream doesn't have it, I would recommend that you experimentally determine the numbers that work well and feel intuitively accurate as well.. just my 2 cents.
